agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -116,14 +116,37 @@ func newPgStatMonitorQAN(q *reform.Querier, dbCloser io.Closer, agentID string,
 	}, nil
 }
 
-func getPGMonitorVersion(q *reform.Querier) (pgMonitorVersion float64, err error) {
+func getPGMonitorVersion(q *reform.Querier) (float64, error) {
 	var v string
-	err = q.QueryRow(fmt.Sprintf("SELECT /* %s */ pg_stat_monitor_version()", queryTag)).Scan(&v)
+	err := q.QueryRow(fmt.Sprintf("SELECT /* %s */ pg_stat_monitor_version()", queryTag)).Scan(&v)
 	if err != nil {
-		return
+		return 0, err
+	}
+
+	return parsePGMonitorVersion(v)
+}
+
+func parsePGMonitorVersion(version string) (float64, error) {
+	split := strings.Split(version, ".")
+	switch len(split) {
+	case 1:
+		return strconv.ParseFloat(split[0], 64)
+	case 2:
+		if len(split[1]) == 1 {
+			split[1] = "0" + split[1]
+		}
```

Comment:
what if version will go up to `0.20.0`?

Reply:
For 0.20.0 it will be 0.2 and for 0.2.0 it will be 0.02. Please check similar cases in test here:
https://github.com/percona/pmm-agent/blob/PMM-8001-pg-stat-monitor-error-handle/agents/postgres/pgstatmonitor/pgstatmonitor_test.go#L481:L487